src/main/java/io/github/oliviercailloux/collaborative_exams/model/entity/question/FreeQuestion.java

```diff
@@ -0,0 +1,213 @@
+package io.github.oliviercailloux.collaborative_exams.model.entity.question;
+
+import java.util.Objects;
+
+import javax.json.bind.annotation.JsonbPropertyOrder;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import io.github.oliviercailloux.collaborative_exams.model.entity.Person;
+
+/**
+ * This Class represents Question
+ * <p>
+ * Question is immuable
+ *
+ * @author Amine BOUKALA
+ */
+@JsonbPropertyOrder({ "id", "author", "phrasing", "language", "type", "isCorrect", "answer" })
+
+@XmlRootElement
+@Entity
+public class FreeQuestion implements IQuestion {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	@XmlAttribute
+	private int id;
+
+	/**
+	 * represent the phrasing of the question
+	 * <p>
+	 * Not <code>null</code>.
+	 */
+	@XmlElement
+	@Column(nullable = false)
+	private final String phrasing;
+
+	/**
+	 * represent the language of Question
+	 * <p>
+	 * <p>
+	 * Not <code>null</code>, may be empty.
+	 */
+	@XmlElement
+	@Column(nullable = false)
+	private String language;
+
+	/**
+	 * respresente the Author
+	 *
+	 * @see Person Not <code>null</code>.
+	 */
+
+	@ManyToOne
+	@XmlElement(name = "author")
+	private Person author;
+
+	/**
+	 * respresente if the response whas correct in T/F or Y/N question
+	 * <p>
+	 * Null if type of question is Free or QCM
+	 */
+	@Column(nullable = true)
+	private boolean isCorrect;
+
+	/**
+	 * Respresent the type of question
+	 *
+	 * @see QuestionType Not <code>null</code>.
+	 */
+	@XmlElement
+	@Column(nullable = false)
+	private QuestionType type;
+
+	/**
+	 * We consider that a free question 
+	 */
+
+	@XmlElement
+	@Column(nullable = false)
+	private Answer answer;
```

Comment:
“We consider that a free question ”… ? Oui ?
Même remarque que pour `isCorrect` : `answer` inapproprié sur `FreeQuestion`.